composer.json

```diff
@@ -24,7 +24,7 @@
     },
     "require-dev": {
         "orchestra/testbench": "^3.8",
-        "phpunit/phpunit": "^8.2"
+        "phpunit/phpunit": "^7.3 || ^8.2"
```

Comment:
This is a fix for https://github.com/nunomaduro/larastan/commit/443f0fe57b55146240e74616e1b42db6fbe92383
In order to have successful unit tests.